### DISCUSSION TITLE
Language

### DISCUSSION THREAD

**Discussion by githubkusi** [OpenCCU/OpenCCU#1429] [🇺🇸 General discussions]
I'm just starting with RaspberryMatic, a really great project. Thanks for all the effort you invested, @jens-maus and supporters. However, I'd appreciate if you enforce the English language at least in the issues. English isn't my native language either, but it's the common technical language which makes it possible to set up an international community. With German, you not only make it more difficult to search for topics/issues but you also exclude people who are not capable of German.

**Comment by jens-maus** [comment]
> I'd appreciate if you enforce the English language at least in the issues. English isn't my native language either, but it's the common technical language which makes it possible to set up an international community.

Thanks for your input, first of all. And I am all with you that english language should be preferably used in documentation, README and also in issue tickets. And as far as my documentation and development efforts are concerned I always try to use english language for gold reasons.

However, as stated in this (https://github.com/jens-maus/RaspberryMatic/issues/981) issue ticket already, I will not enforce or pressure users calling out for help or submitting an issue to this project to only use english, simply because the whole homematic ecosystem is mostly used in german speaking countries so far and the number of users not being able to follow the development would be too large IMHO. So I still try to accept german contributions here+there (issues, docs, etc.) but indeed prefer to use english language wherever possible.

And if there are some english resources missing (e.g. in the documentation/wiki) feel free to contribute english translations as a contribution to this project!

**Reply by githubkusi** [reply]
Thanks for the detailed answer. I currently don't see a reason why Homematic couldn't be used in any country with 220V. But indeed you're right, I couldn't find web shops in the English language selling homematic gear.
Anyways, whenever I stumble upon German docs during my journey, I'll help to translate it to English :)
